src/xenia/kernel/xboxkrnl/xboxkrnl_io.cc

```diff
@@ -105,6 +105,12 @@ dword_result_t NtCreateFile(lpdword_t handle_out, dword_t desired_access,
     return X_STATUS_OBJECT_NAME_INVALID;
   }
 
+  std::error_code directory_error_code;
+  if (std::filesystem::is_directory(target_path, directory_error_code) &&
```

Comment:
Will this work with virtual file systems? I think is_directory should ask the host OS, not just check the path.

Reply:
Yeah, it shouldn't be touching the real FS at all. Should be an invoke into the VFS.

Reply:
I agree. I didn't factor that it touches real file system